tests/test_harness.rs

```diff
@@ -100,7 +100,7 @@ impl Harness {
 
         for dir_entry in dir_iter.into_iter() {
             let current_path = dir_entry?.path();
-            if current_path.extension() == extension.as_deref() {
+            if current_path.extension() == extension.as_deref() && current_path.is_file() {
```

Comment:
Consider https://rust-lang.github.io/rust-clippy/master/index.html#filetype_is_file

Reply:
Fixed in f90c6b4